source/material_model/melt_simple.cc

```diff
@@ -39,7 +37,7 @@ namespace aspect
     reference_darcy_coefficient () const
     {
       // 0.01 = 1% melt
-      return reference_permeability * std::pow(0.01,3.0) / eta_f;
+      return katz2003_model.get_reference_permeability() * std::pow(0.01,3.0) / katz2003_model.get_eta_f();
```

Comment:
I did that because I thought `reference_darcy_coefficient` was a required function for `MeltInterface` which `MeltSimple` extends. I could make another `reference_darcy_coefficient` in the Katz2003 class which the function in `MeltSimple` calls, but I just broke it up into getters for each variable. Would making the identical function in the Katz2003 class be clearer?

Reply:
My vote would be to make an identical function in the Katz2003 class, as this will be needed regardless for extensions to other material models?